R/record_versions.R

```diff
@@ -32,7 +28,7 @@ record_versions <- function(
   previous <- read_versions_previous(manifest = manifest)
   new <- update_version_manifest(current = current, previous = previous)
   jsonlite::write_json(x = new, path = manifest, pretty = TRUE)
-  aligned <- versions_aligned(manifest = new, check_hash = check_hash)
+  aligned <- versions_aligned(manifest = new)
```

Comment:
Given `versions_aligned()` is simplified with only one argument, I would find it easier to follow if it was just inlined here.